EIPS/eip-5202.md

```diff
@@ -30,6 +33,7 @@ A blueprint contract MUST contain at least one byte of initcode.
 A blueprint contract MAY insert any bytes (data or code) between the version byte(s) and the initcode. If such variable length data is used, the preamble must be `0xFE71<version bits><length encoding bits><length bytes><data>`. The `<length encoding bits>` represent a number between 0 and 2 (inclusive) describing how many bytes `<length bytes>` takes, and `<length bytes>` is the big-endian encoding of the number of bytes that `<data>` takes.
 
```

Comment:
I'd like to see an example of a blueprint contract with and also without a data section.

Reply:
added some examples in 675b3e820aaac5b93c32683c747ee301f07aad55